index.html

```diff
@@ -4,13 +4,163 @@
+        <li><img class="toolbar-logo" src="./Images/facebook-svgrepo-com.svg" alt="Facebook"></li>
+        <li><img class="toolbar-logo" src="./Images/twitter-svgrepo-com.svg" alt="Twitter"></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">Mypage</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <nav class="nav-bar">
+      <div class="mobile"><img class="hamburguer-menu" src="./Images/Hamburguer_Menu.png"/></div>
+      <a href="index.html"><img class="forum-logo desktop" src="./Images/Forum Logo.png"/></a>
+      <ul class="navbar-menu-list desktop">
+        <li><a class="nav-bar-menu-li" href="about.html">About</a></li>
+        <li><a class="nav-bar-menu-li" href="#program">Program</a></li>
+        <li><a class="nav-bar-menu-li"href="#">Join</a></li>
+        <li><a class="nav-bar-menu-li">Sponsor</a></li>
+        <li><a class="nav-bar-menu-li" href="#">News</a></li>
+        <li><a class="gef-campaing" href="#">GEF Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section class="home-intro">
+      <div class="overlay-gray-home"></div>
+      <div class="bg-image-home"></div>
+      <div class="home-data-container">
+        <h2>"Protect your future enviroment!!"</h2>
+        <h1>CONSERVATION GLOBAL FORUM 2023</h1>
+        <div>  
+          <p class="home-intro-parag-container">A massive kickoff where the most respectable Sustainability Desginers will define a Global plan to protect our nature.</p>
+        </div>  
+        <div class="home-intro-location-container">
+          <h2>2023.05.24(WED) ~ 26(FRI)</h2>
+          <p>@ Chapelco Golf Resort, San Martín de Los Andes, Argentine Patagonia. </p>
+        </div>
+      </div>
+    </section>
+
+    <section id="program" class="main-program">
+      <h2>Main Program</h2>
+      <div class="separator"></div>
+
+      <div class="all-activities-container">
+        <div class="activity-container">
+          <div class="activity">
+          <div class="icon-container">
+            <img class="program-icon" src="./Images/Icons/planet-earth-svgrepo-com.svg" alt=""/>
+            <span>Sponsors</span>
+          </div>
+          <p class="activity-paragraph">Know what enviromental care companies are doing about restoration.</p>
+        </div>
+      </div>  
+
+      <div class="activity-container">
+        <div class="activity">
+          <div>
+            <img class="program-icon" src="./Images/Icons/document-envelope-envelope-general-svgrepo-com.svg" alt=""/>
+            <span>Projects</span>
+          </div>
+          <p>Appreciate various proteccionism parks and landscapes recovered using private companies profits.</p>
+        </div>
+      </div>  
+
+      <div class="activity-container">
+        <div class="activity">
+          <div>
+            <img class="program-icon" src="./Images/Icons/average-image-image-images-svgrepo-com.svg" alt=""/>
+            <span>Brand Value</span>
```

Comment:
- [ ] As per the previous reviewer's suggestion, it is important to add an `alt` attribute to all the images on your page. This **will ensure that your page is accessible to everyone**, including individuals who **may rely on screen readers or have visual impairments.** Please remember to include descriptive alternative text for each image to provide a meaningful description of the image content. 🙋‍♂️